.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 repos:
 -   repo: https://github.com/phantomcyber/dev-cicd-tools
-    rev: v1.17
+    rev: v1.23
```

Comment:
There is no changes in unreleased.md. Is that intended? Does this PR require changelog?